frontend/src/bundles/admin-panel/mock-data/mock-data.ts

```diff
@@ -4,206 +4,210 @@ const placeHolder = 'This is just a placeholder name';
+        fullName: placeHolder,
     },
     {
         userId: 'employer_4',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_5',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_6',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_7',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_8',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_9',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_10',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_11',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_12',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_13',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_14',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_15',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_16',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_17',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_18',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_19',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
     {
         userId: 'employer_20',
-        avatar: loremPicsum,
-        username: placeHolder,
+        photoUrl: loremPicsum,
+        fullName: placeHolder,
     },
 ];
 
```

Comment:
Do we need mock data ?

Reply:
As for me, in this PR you need to change mock data for admin to api calling ?

Reply:
It still used on another page

Reply:
where? why we can not remove it?